### ISSUE TITLE
Matplotlib resizable

### ISSUE THREAD

**Issue by TomVeniat** [facebookresearch/visdom#656]
**Bug Description**
When calling `visdom.matplot()` with the `resizable` option set to True, I obtain the following error:
```
Traceback (most recent call last):
  File "/Users/veniat/src/visdom/example/demo.py", line 562, in <module>
    Visdom().matplot(plt, opts={'resizable': True})
  File "/Users/veniat/src/visdom/py/visdom/__init__.py", line 1028, in matplot
    opts['height'] = 1.4 * int(math.ceil(float(height)))
ValueError: could not convert string to float: '345.6pt'
```
It seems that when the `resizable` flag is True, the string returned by bs4 for the height and width attributes will be in the format `"123.4pt"` rather than the expected `"123.4"`. 
I tried with matplotlib 3.0 and 3.1 and the error happens in both cases.

**Reproduction Steps**
Add the `opts={'resizable': True}` attribute to [matplot call](https://github.com/facebookresearch/visdom/blob/1cb8c19cb9fb86b3930ab06ce6d19837fba81244/example/demo.py#L57) in demo.py

One simple solution would be to add something like 
``` python      
if height and height.endswith('pt'):
    height = height[:-2]
```
After getting the `width` and `height` attributes with bs4, keeping the current behaviour if the string is in the expected fomat.

**Comment by JackUrb** [comment]
Ah this is indeed a bug - thanks for the flag @TomVeniat. That solution seems like it would work, happy to accept a PR, otherwise it'll be on the backlog for a little.

**Comment by JackUrb** [comment]
Should be fixed as of #710!
